common/src/test/java/gov/cms/ab2d/common/util/AB2DPostgresqlContainer.java

```diff
@@ -1,24 +1,70 @@
 package gov.cms.ab2d.common.util;
 
 import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.images.builder.ImageFromDockerfile;
+import org.testcontainers.utility.DockerImageName;
 
+import java.io.IOException;
 import java.util.Collections;
 
 public class AB2DPostgresqlContainer extends PostgreSQLContainer<AB2DPostgresqlContainer> {
 
-    private static final String IMAGE_VERSION = "postgres:11";
+    private static final ImageFromDockerfile DOCKER_IMAGE = new ImageFromDockerfile()
+        .withDockerfileFromBuilder(builder ->
+            builder
+                .from("postgres:15.2")
+                .run("apt-get -y update")
+                .run("touch /docker-entrypoint-initdb.d/init_pg_cron.sh")
+                .run("echo \"#!/bin/sh\"\n >> /docker-entrypoint-initdb.d/init_pg_cron.sh")
+                .run("echo \"# Configure PG Cron\"\n >> /docker-entrypoint-initdb.d/init_pg_cron.sh")
+                .run("echo \"echo shared_preload_libraries = 'pg_cron' >> /var/lib/postgresql/data/postgresql.conf\"\n >> /docker-entrypoint-initdb.d/init_pg_cron.sh")
+                .run("echo \"echo cron.database_name = 'test' >> /var/lib/postgresql/data/postgresql.conf\"\n >> /docker-entrypoint-initdb.d/init_pg_cron.sh")
+                .run("echo \"# Required to load pg_cron\"\n >> /docker-entrypoint-initdb.d/init_pg_cron.sh")
+                .run("echo \"pg_ctl restart\"\n >> /docker-entrypoint-initdb.d/init_pg_cron.sh")
+                .run("chmod +x /docker-entrypoint-initdb.d/init_pg_cron.sh")
+                .run("apt-get install -y curl")
+                .run("apt-get install -y postgresql-15-cron")
```

Comment:
Can these lines be moved to the postgres Dockerfile, then that file be referenced with withFileFromPath?

Reply:
I am not sure. I originally attempted to have init_pg_cron.sh already exist inside the repo, and then just copy it to the container with a:

.copy("<path_to_init_pg_cron.sh>", "/docker-entrypoint-initdb.d/");

The problem I ran into was a null pointer exception with reference to the init_pg_cron.sh path. I tried both a relative path and an absolute path but neither worked. I am probably missing something, but for testing purposes I gave up and just created the script inside the container on the fly using the commands above.